pyproject.toml

```diff
@@ -1,8 +1,12 @@
 [tool.poetry]
 name = "ftdc-tools"
 version = "0.1.0"
-description = ""
-authors = ["Mridul Augustine <mridul.augustine@mongodb.com>"]
+description = "Pure python package that provides tools required to decode and process FTDC data."
+authors = ["Mridul Augustine <mridul.augustine@mongodb.com>",
```

Comment:
Should this include dev prod tips with the devprod tips email? I think `devprod-tips@mongodb.com `